apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts

```diff
@@ -373,6 +373,7 @@ export class PrivateFlightController {
   }
 
   @Get('users/:nationalId/flights')
+  @Scopes(AirDiscountSchemeScope.admin, AirDiscountSchemeScope.default)
```

Comment:
Should refactor this controller like I did with PrivateDiscountController, so we have `PrivateFlightController` with this endpoint, and then `PrivateFlightAdminController` with the other endpoints. Then you can have different ApiTags on them and avoid depending on the Admin API in the discount resolver.

Reply:
Oh yeah, good point. I incorrectly assumed the endpoint was being used.
Will do :+1:

Reply:
Done